.github/workflows/publish.yaml

```diff
@@ -0,0 +1,24 @@
+name: Publish package
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Install mise
+        uses: jdx/mise-action@v2
+      - run: npm ci
+      - run: npm run build
+      - name: Publish package
+        uses: JS-DevTools/npm-publish@v3
```

Comment:
I don't know if this is really what we want - it seems like a tool with pretty low usage and kind of weak provenance, and using manual version-number updates is prone to mistakes.

I would be much more amenable to something like https://github.com/googleapis/release-please or doing manual releases with `npx np`, which is how we're releasing versions of all of our other packages.